pkg/gencmd/command.go

```diff
@@ -0,0 +1,77 @@
+package gencmd
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type CommandType int
+
+const (
+	CommandUnknown CommandType = iota
+	CommandGenerate
+	CommandDestroy
+)
+
+type Command struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Command should have comment or be unexported